_includes/carousel.html

```diff
@@ -0,0 +1,222 @@
+{% assign letterstring = "a,b,c,d,e,f,g,h,i,j,k,l,m,n" %}
+{% assign letters = letterstring | split: ',' %}
+
+<div class="carousel__holder" style="margin-top: 0px; padding-top: 0px;">
+    <div class="carousel">
+        {% for item in site.data.carousel.images %}
+          <input class="carousel__activator" type="radio" name="carousel" id="{{ letters[forloop.index0] }}" {% if forloop.first %}checked="checked"{% endif %} />
+        {% endfor %}
+        {% for item in site.data.carousel.images %}
```

Comment:
this syntax is so painful, lol

Reply:
straight copypasta from thing on github didn't even read